core/src/main/java/com/linecorp/armeria/common/multipart/MultiPartDecoder.java

```diff
@@ -54,7 +54,7 @@
 /**
  * Reactive processor that decodes HTTP payload as a stream of {@link BodyPart}.
  */
-public class MultiPartDecoder implements Processor<HttpData, BodyPart> {
+class MultiPartDecoder implements Processor<HttpData, BodyPart> {
```

Comment:
- MultiPart -> Multipart :wink:
- `final`?

Reply:
The subscriber methods of MultiPartDecoder are overridden by `MultipartDecoderSubsWhiteBoxTckTest` for TCK.